sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -700,7 +700,7 @@ private void processMinidumpFiles() {
 
                 @Override
                 public boolean accept(File dir, String filename) {
-                    return !filename.equals(DEVICE_INFO_FILE);
+                    return filename.endsWith(".dmp");
```

Comment:
Let's move the extension to ErrorLogHelper constants with a comment like we did for json:
https://github.com/microsoft/appcenter-sdk-android/blob/develop/sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java#L61

Reply:
Done